JS/Prototypes/PrototypeAttribute.html

```diff
@@ -11,32 +11,41 @@
 
 
 <script>
+// function Items(){
+//     this.fruit="Mango";
+// }
+
+// //here  we created property veggie  for items with  prototype which has access to all that created by items()
+//  Items.prototype.veggie ="tomato"
+
+// var sweetFruit =new Items();
+
+// sweetFruit.fruit="Straw Berry"
+    
+
+
+// console.log(sweetFruit.fruit);//here fruit look first in Items object and then to new defined sweetfruit prints  gives "Straw Berry "in console
+// console.log(sweetFruit.veggie);//here all the access of items will be accesed by sweetFruit so that it gives tomato  
+                                              
+
+
+// console.log(sweetFruit.toString());
+// // in this all items object inherited all properties inherited by object.prototype
+ 
 function Items(){
     this.fruit="Mango";
 }
-
 //here  we created property veggie  for items with  prototype which has access to all that created by items()
  Items.prototype.veggie ="tomato"
```

Comment:
difference between adding a property on the prototype chain vs at the function level?

Reply:
with constructor(function level):   Every object has its own instance of the function
with the prototype:   Modifying a property using one object reflects the other object also

I also added one js file to showing differences with both approaches